cmd/frontend/graphqlbackend/graphqlbackend.go

```diff
@@ -90,6 +93,23 @@ VARIABLES
 		if finish != nil {
 			finish(err)
 		}
+		d := time.Since(start)
+		if v := conf.Get().ObservabilityLogSlowGraphQLRequests; v != 0 && d.Milliseconds() > int64(v) {
```

Comment:
Can `v` ever be negative syntactically? If so, suggest 
```suggestion
		if v := conf.Get().ObservabilityLogSlowGraphQLRequests; v > 0 && d.Milliseconds() > int64(v) {
```

Same for the other place.

Reply:
cc @slimsag in case you won't revisit this PR again.